modules/tinymce/src/themes/silver/test/ts/module/StickyHeaderUtils.ts

```diff
@@ -184,6 +185,19 @@ const assertEditorClasses = (docked: boolean) => {
   })), container);
 };
 
+const assertHeaderPosition = async (toolbarLocation: ToolbarLocation, value: number) => {
```

Comment:
I missed this previously sorry, but as this is async it should use the `p` prefix.

- [ ] rename this to `pAssertHeaderPosition` to keep consistent